src/Bicep.Wasm/LanguageHelpers/SemanticTokenVisitor.cs

```diff
@@ -271,5 +271,14 @@ public override void VisitTargetScopeSyntax(TargetScopeSyntax syntax)
             AddTokenType(syntax.Keyword, SemanticTokenType.Keyword);
             base.VisitTargetScopeSyntax(syntax);
         }
+
+        public override void VisitImportDeclarationSyntax(ImportDeclarationSyntax syntax)
+        {
+            AddTokenType(syntax.Keyword, SemanticTokenType.Keyword);
+            AddTokenType(syntax.AliasName, SemanticTokenType.Variable);
+            AddTokenType(syntax.FromKeyword, SemanticTokenType.Keyword);
```

Comment:
>FromKeyword [](http://example.com/codeflow?start=32&length=11)

Should this be using `AddContextualKeyword` instead? (If you typo the `from` keyword, does it get the keyword highlighting?)

Reply:
Good point, fixed.